iothub/service/src/FeedbackBatch.cs

```diff
@@ -11,10 +11,19 @@ namespace Microsoft.Azure.Devices
     /// </summary>
     public class FeedbackBatch
     {
+        /// <summary>
+        /// A timestamp that indicates when the feedback message was received by the IoT hub.
```

Comment:
The concept of feedback message is probably not always clear. Can we include some context of what a feedback message is with each mention? Or refer to a central location where a feedback message is defined?